src/crimson/os/seastore/btree/fixed_kv_node.h

```diff
@@ -1002,6 +1006,10 @@ struct FixedKVLeafNode
   FixedKVLeafNode(ceph::bufferptr &&ptr)
     : FixedKVNode<NODE_KEY>(has_children ? CAPACITY : 0, std::move(ptr)),
       node_layout_t(this->get_bptr().c_str()) {}
+  FixedKVLeafNode(extent_len_t length)
+    : FixedKVNode<NODE_KEY>(has_children ? CAPACITY : 0,
+        ceph::bufferptr(buffer::create_page_aligned(length))),
+      node_layout_t(this->get_bptr().c_str()) {}
```

Comment:
This commit looks unnecessary, probably easier to understand if squash into the commit having its actual user.